### ISSUE TITLE
Bug - On submitted forms,  Answers and File uploads are not showing in the app or portal.

### ISSUE THREAD

**Issue by devmatics-issues** [devmatics/clockwork-issues#19]
This is going to combine multiple reported bugs into one since I believe they are all related.

**Master Loom: Tested All problems at once**
https://www.loom.com/share/e28411c9fc7842a3abc7e5d06f433f0e

# Details: 1
**Category:** Bug 1: Date is not displaying in submitted report history in app and occasionally in portal.

**Short Description:** Date is not displaying in submitted report history.

**Full Description:**
Dates do not appear for submitted forms.

**Loom Reproduced Error:**
https://www.loom.com/share/56e056e2de274b32b32adc74c740cb39

**Attachments:**
![](https://www.clockworksafety.com/wp-content/uploads/formidable/4/chrome_wrwQ1abYWz.png)

**Priority:** Medium
**User Type:** Employee (App)
**Device:** Desktop - Windows
**Browser:** Google Chrome
**Source:** https://www.clockworksafety.com/support-ticket/

**Submitted By:**
Clockwork Safety (admin@devmatics.io)

# Details:2
**Category:** Bug 2: Multiple choice responses (and probably single responses) do not appear in responses history for portal users.

**Short Description:** Multiple choice responses (and probably single responses) do not appear in  responses history for portal users.

**Full Description:**
Multiple Choice responses and Single-select response do not display an answer when looking at a submitted form.

**Attachments:**
![](https://www.clockworksafety.com/wp-content/uploads/formidable/4/chrome_p5S9Tn3m3l.png)

**Priority:** Medium
**User Type:** Manager (Portal)
**Device:** Desktop - Windows
**Browser:** Google Chrome
**Source:** https://www.clockworksafety.com/support-ticket/

**Submitted By:**
Clockwork Safety (admin@devmatics.io)

# Details: 3
**Category:** Bug Uploaded files do not appear on submitted forms for portal users.

**Short Description:** Uploaded files do not appear on submitted forms for portal users.

**Full Description:**
I have seen this bug before, but I was unable to reproduce it in today's (11/10/2021) test

**Attachments:**
![](https://www.clockworksafety.com/wp-content/uploads/formidable/4/chrome_YAaL3Ew6M6.png)

**Priority:** Medium
**User Type:** Manager (Portal)
**Device:** Desktop - Windows
**Browser:** Google Chrome
**Source:** https://www.clockworksafety.com/support-ticket/

**Submitted By:**
Clockwork Safety (admin@devmatics.io)

# Details: 4
**Category:** Bug: Uploaded files do not appear on submitted forms for App users.

**Short Description:** Uploaded files do not appear on submitted forms for App users.

**Full Description:**
I have seen this bug before, but I was unable to reproduce it in today's (11/10/2021) test

**Attachments:**
![](https://www.clockworksafety.com/wp-content/uploads/formidable/4/chrome_mVxecaPRuS.png)

**Priority:** Medium
**User Type:** Employee (App)
**Device:** Desktop - Windows
**Browser:** Google Chrome
**Source:** https://www.clockworksafety.com/support-ticket/

**Submitted By:**
Clockwork Safety (admin@devmatics.io)

# Related Issues

- [x] https://github.com/devmatics/clockwork-portal/issues/42
- [x] https://github.com/devmatics/clockwork-app/issues/25

**Comment by ARegalbuto** [comment]
@alexdwhite This bug presented a view different issues when I tested it.  I tested it once while recording a HAR file.  It was strange enough I reproduced it again with a HAR file and a Loom.
[bug 19 HAR.zip](https://github.com/devmatics/clockwork-issues/files/7516758/bug.19.HAR.zip)
https://loom.com/share/56e056e2de274b32b32adc74c740cb39

I feel this may be an api issue, or like what Yassine was saying that the forms are being miss-assigned in app?
